oremda/registry.py

```diff
@@ -27,38 +27,27 @@ def __init__(self, memory_client: MemoryClient, container_client: ContainerClien
 
     def _inspect(self, image_name: str):
         image: Any = self.container_client.image(image_name)
-
-        labels = image.oremda_labels
-
-        ports = labels.setdefault('ports', {})
-        ports.setdefault('input', {})
-        ports.setdefault('output', {})
-
-        name = labels.get('name')
-
-        assert(name is not None)
-
-        return labels
```

Comment:
This body of code was removed because the ports and name are now guaranteed to be there due to pydantic.